src/common/com/intellij/plugins/haxe/ide/info/HaxeParameterInfoHandler.java

```diff
@@ -80,11 +80,13 @@ public void showParameterInfo(@NotNull PsiElement element, CreateParameterInfoCo
 
   @Nullable
   private static HaxeFunctionDescription tryGetDescription(HaxeCallExpression callExpression) {
-    final PsiElement target = ((HaxeReference)callExpression.getExpression()).resolve();
+    final HaxeReference expression = (HaxeReference)callExpression.getExpression();
+    final PsiElement target = expression.resolve();
+    final boolean isExtension = expression.resolveIsExtension();
```

Comment:
Hmm.  Took me a little while to figure out that isExtension everywhere really means is a static extension.  is*Static*Extension would be a better name.  Not a big deal, though.